Userland/Applications/BrowserSettings/Defaults.h

```diff
@@ -10,8 +10,8 @@
 
 namespace Browser {
 
-static constexpr StringView default_homepage_url = "resource://html/misc/welcome.html"sv;
-static constexpr StringView default_new_tab_url = "resource://ladybird/new-tab.html"sv;
+static constexpr StringView default_homepage_url = "about:newtab"sv;
```

Comment:
Not sure that we want to be changing the welcome page quite yet. Eventually it should go away, as all of its test pages are changed to real LibWeb tests. But I don't think we're there yet.